sdk/core/Azure.Core/src/Diagnostics/AzureCoreEventSource.cs

```diff
@@ -12,29 +12,12 @@ namespace Azure.Core.Diagnostics
     [EventSource(Name = EventSourceName)]
     internal sealed class AzureCoreEventSource : AzureEventSource
     {
-        private const string EventSourceName = "Azure-Core";
+        private const string EventSourceName = "Azure-Core-ext";
```

Comment:
This will still work with AzureCoreEventListener while avoiding conflicts with the event source created for "Azure-Core" created inside of System.ClientModel. The downside is that anyone with filtering that they implement on their own in something like app insights could be broken if they do an equals check on the name itself rather than using traits or doing something like a contains/starts with check.